package.json

```diff
@@ -275,6 +310,28 @@
           "command": "one.toolchain.uninstall",
           "when": "view == ToolchainView && viewItem =~ /toolchain/ && !one.job:running",
           "group": "inline"
+        },
+        {
+          "command": "one.viewer.metadata.show",
+          "when": "view == OneExplorerView && viewItem =~ /baseModel|product/",
+          "group": "7_metadata@1"
+        },
+        {
+          "command": "one.viewer.relation.show",
+          "when": "view == OneExplorerView && viewItem =~ /baseModel|product/",
+          "group": "7_metadata@2"
+        }
+      ],
+      "explorer/context": [
+        {
+          "command": "one.viewer.metadata.show",
+          "when": "resourceExtname in one.metadata.supportedFiles && !explorerResourceIsFolder",
+          "group": "7_metadata@1"
+        },
+        {
+          "command": "one.viewer.relation.show",
```

Comment:
Please add those commands to "commandPalette" and set "false".

For example, L399 of `package.json`,
![image](https://user-images.githubusercontent.com/17171963/193398116-df0a3cab-e0ff-4aab-abd2-e83b4ad96794.png)


This will conceal those commands from command palette.
Currently, they are shown as below. 
![image](https://user-images.githubusercontent.com/17171963/193398237-9c397939-cf52-4923-a690-0e86d279f689.png)

Reply:
ok, I`ll apply that part.